ngx-fudis/projects/ngx-fudis/src/lib/components/form/guidance/guidance.component.html

```diff
@@ -104,6 +106,7 @@
     <ng-container *ngFor="let singleControl of formGroup?.controls | keyvalue">
       <small
         *ngIf="singleControl.key === selectedOption"
+        aria-hidden="true"
```

Comment:
Remove this aria-hidden and add visually hidden text after the maxLength indicator, like so:

```
        {{ singleControl.value.value?.length || 0 }}/{{ maxLength
        }}<span class="fudis-visually-hidden">{{ _maxLengthText | async }}</span>
```